test/e2e/edu-helpers.js

```diff
@@ -18,26 +18,47 @@ function initApplicationSubmitMock(form) {
   });
 }
 
-function completeVeteranInformation(client, data, onlyRequiredFields) {
+function completeVeteranInformation(client, data, onlyRequiredFields, root = 'root') {
+  // Use relativeFullname if necessary
```

Comment:
I don't get when we would want to use the relative data to fill the veteran info.